### PR TITLE
Install collection to local directory during development

### PR DESCRIPTION
**Pull request by l3ender**
##### SUMMARY
This PR configures the collection so that it is installed to the project's directory when doing local development/testing. This allows for better isolation when doing development/testing of changes. The current install location is a system-wide location which affects all other Ansible work done on the same system.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
development

##### ADDITIONAL INFORMATION
When doing development in this repo, I run the following command:
```bash
rm -f azure-azcollection-*.tar.gz && ansible-galaxy collection build . --force && ansible-galaxy collection install azure-azcollection-*.tar.gz --force
```

Output of command BEFORE this change:
```bash
Created collection for azure.azcollection at /Users/ross/repos/azure/azure-azcollection-1.11.0.tar.gz
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'azure.azcollection:1.11.0' to '/Users/ross/.ansible/collections/ansible_collections/azure/azcollection'
azure.azcollection:1.11.0 was installed successfully
```

Output of command AFTER this change:
```bash
Created collection for azure.azcollection at /Users/ross/repos/azure/azure-azcollection-1.11.0.tar.gz
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'azure.azcollection:1.11.0' to '/Users/ross/repos/azure/ansible_collections/azure/azcollection'
azure.azcollection:1.11.0 was installed successfully
```

### CODE REVIEW

**Comment by l3ender** [comment]
@Fred-sun @xuzhang3 Can you please take a look at this PR? I think it will help minimize conflicts during development. Thank you!!

**Comment by Fred-sun** [comment]
> @Fred-sun @xuzhang3 Can you please take a look at this PR? I think it will help minimize conflicts during development. Thank you!!

@l3ender Thank you for your contribution. Could you please explain why you made the above changes? How does this avoid conflict? Will facilitate the consolidation of this PR. Thank you very much!

**Comment by l3ender** [comment]
@Fred-sun Installing the collection to a project-specific path allows for better isolation of multiple projects when doing development/testing of changes. For example, a person could have multiple different projects which utilize this collection. When doing development on this collection and installing it to the default location, it then can break all other projects which use this collection.

By implementing the PR to install the collection to a project-specific location, it will prevent any conflict or error arising from installing the collection to the global location and breaking other items using azcollection.

Hope that helps explain the use case. Thank you!

**Comment by l3ender** [comment]
Hi @xuzhang3, could you please take a look at this one? Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ When contributing to this repository, please first discuss the change you wish t
 
 1. Build/install the collection:
     ```bash
-    rm azure-azcollection-*.tar.gz && ansible-galaxy collection build . --force && ansible-galaxy collection install azure-azcollection-*.tar.gz -p tests/staging --force
+    rm -f azure-azcollection-*.tar.gz && ansible-galaxy collection build . --force && ansible-galaxy collection install azure-azcollection-*.tar.gz -p tests/staging --force
```
Comment (xuzhang3) [inline CONTRIBUTING.md]: since we have set the collection install location in the `ansible.cfg`,`-p tests/staging`  should be removed

**Review by xuzhang3 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (l3ender) [reply CONTRIBUTING.md]: Good catch, @xuzhang3...thank you! I have updated it.

**Comment by xuzhang3** [comment]
LGTM， shipping it 🛥️

**Pull request merged**